Add support for adding subcategories description #1098 (#1143)
rupali-codes/LinksHub: CONTRIBUTING.md

---

### Adding and Updating Category Description 📝

> > ✨ You can also create [issue(s)](https://github.com/rupali-codes/LinksHub/issues/new/choose) to suggest new descriptions, and someone else will take care of them.

Reviewer: [Proofreading]
```suggestion
> ✨ You can also create [issue(s)](https://github.com/rupali-codes/LinksHub/issues/new/choose) to suggest new descriptions. The maintainers will review them and then assign them.
```

#### 🌟Follow these steps to add your links to the Hub

- Choose the category and subcategory under which you want to add a description.
- Locate the file: `components/TopBar/CategoryDescriptions.ts`
- Add or update the description in front of the subcategory name.

>   > For example, to add description of `fonts` subcategory under `frontend` categrogy

Reviewer: [Proofreading] 
```suggestion
  > For example, to add the description of the `fonts` subcategory under the `frontend` category, you would do the following: 
```
